include/wx/dvrenderers.h

```diff
@@ -317,9 +317,6 @@ class WXDLLIMPEXP_CORE wxDataViewCustomRendererBase
     // Render the item using the current value (returned by GetValue()).
     virtual bool Render(wxRect cell, wxDC *dc, int state) = 0;
 
-    // Return the size of the item appropriate to its current value.
-    virtual wxSize GetSize() const = 0;
-
```

Comment:
Why did you remove this? This (expectedly) broke the builds everywhere but macOS, yet you somehow don't seem at all to be bothered by this... I just don't understand your thinking here.